docs/remote/containers.md

```diff
@@ -117,20 +111,39 @@ The steps are similar to those above:
 
-3. After picking the starting point for your container, VS Code will add the dev container configuration files to your project (`.devcontainer/devcontainer.json`). The VS Code window will reload and start building the dev container. A progress notification provides you status updates. Note that you only have to build a dev container the first time you open it; opening the folder after the first successful build will be much quicker.
+3. After picking the starting point for your container, VS Code will add the dev container configuration files to your project (`.devcontainer/devcontainer.json`).
+
+4. The VS Code window will reload and start building the dev container. A progress notification provides you status updates. Note that you only have to build a dev container the first time you open it; opening the folder after the first successful build will be much quicker.
 
     ![Dev Container Progress Notification](images/containers/dev-container-progress.png)
 
-4. After building completes, VS Code will automatically connect to the container. You can now interact with your project in VS Code just as you could when opening the project locally. From now on, when you open the project folder, VS Code will automatically pick up and reuse your dev container configuration.
+5. After the build completes, VS Code will automatically connect to the container.
+
+You can now interact with your project in VS Code just as you could when opening the project locally. From now on, when you open the project folder, VS Code will automatically pick up and reuse your dev container configuration.
 
 > **Tip:** Want to use a remote Docker host? See the [Advanced Containers article](/docs/remote/containers-advanced.md#developing-inside-a-container-on-a-remote-docker-host) for details on setup.
 
+### Open an existing workspace in a container
+
+You can also follow a similar process to open a [VS Code multi-root workspace](/docs/editor/multi-root-workspaces) in a **single container** using the **Remote-Containers: Open Workspace in Container...** command. After selecting a `.code-workspace` file, a `.devcontainer` folder will be added next to the workspace file (if one is not already present).
+
+Alternatively, you can also simply use **File > Open Worksapce...** once you've opened a folder that contains a `.code-workspace` file in a container.
+
+For the workspace to load completely, all referenced paths will need to either be mounted or coped into the container. Therefore, the workspace must either:
+
+* Reference relative paths to sub-folders of the folder the `.code-workspace` file is in (or the folder itself).
+* Reference paths that have been added to the container as [custom mount points](/docs/remote/containers-advanced.md#adding-another-local-file-mount).
```

Comment:
We currently check that all folders are relative paths that exist on the **local filesystem** before we open the workspace in the container. That limits the options here to the first.

Reply:
Currently that is not allowed.